web3_proxy/src/frontend/users/payment.rs

```diff
@@ -236,7 +237,7 @@ pub async fn user_balance_post(
 
     // check for uncles
     let mut find_uncles = increase_on_chain_balance_receipt::Entity::find()
-        // .lock_exclusive()
```

Comment:
i'm not sure if we want this, but it seems like it should have a lock

Reply:
Sounds about right.